### ISSUE TITLE
nicer way to report errors during init?

### ISSUE THREAD

**Issue by nealio82** [mnapoli/bref#322]
When running `bref init`, the process exited immediately and with a not-particularly-helpful exception message (`The process has been signaled with signal "6".`) coming from `symfony/process`.

I had to re-run with `-v` to find out where it was going wrong (the AWS configure step), and then try running the command myself just to see what's happening.

It might be nicer if we can catch some of these and give the information back to the user in a friendlier way

```zsh
➜  bref-function$ vendor/bin/bref init

In Process.php line 426:

  The process has been signaled with signal "6".


init

➜  bref-function$ vendor/bin/bref init -v

In Process.php line 426:

  [Symfony\Component\Process\Exception\ProcessSignaledException]
  The process has been signaled with signal "6".


Exception trace:
 () at /Users/neal/Projects/bref-function/vendor/symfony/process/Process.php:426
 Symfony\Component\Process\Process->wait() at /Users/neal/Projects/bref-function/vendor/symfony/process/Process.php:235
 Symfony\Component\Process\Process->run() at /Users/neal/Projects/bref-function/vendor/mnapoli/bref/bref:39
 Silly\Application->{closure}() at n/a:n/a
 call_user_func_array() at /Users/neal/Projects/bref-function/vendor/php-di/invoker/src/Invoker.php:82
 Invoker\Invoker->call() at /Users/neal/Projects/bref-function/vendor/mnapoli/silly/src/Application.php:95
 Silly\Application->Silly\{closure}() at /Users/neal/Projects/bref-function/vendor/symfony/console/Command/Command.php:253
 Symfony\Component\Console\Command\Command->run() at /Users/neal/Projects/bref-function/vendor/symfony/console/Application.php:908
 Symfony\Component\Console\Application->doRunCommand() at /Users/neal/Projects/bref-function/vendor/symfony/console/Application.php:269
 Symfony\Component\Console\Application->doRun() at /Users/neal/Projects/bref-function/vendor/symfony/console/Application.php:145
 Symfony\Component\Console\Application->run() at /Users/neal/Projects/bref-function/vendor/mnapoli/bref/bref:160

init

➜  bref-function$ aws configure
dyld: Library not loaded: @executable_path/../.Python
  Referenced from: /usr/local/aws/bin/python2.7
  Reason: image not found
[1]    44135 abort      aws configure
```

**Comment by mnapoli** [comment]
:+1:

By the way is the `aws configure` error caused by something we should fix as well?

**Comment by nealio82** [comment]
nope, that was caused by a broken python install!

**Comment by mnapoli** [comment]
Fixed by #323
